src/Tickets/Emails/Admin/Preview_Modal.php

```diff
@@ -240,42 +240,36 @@ public function get_modal_content_ajax( $render_response, $vars ) {
 			$preview_context['add_qr_codes'] = tribe_is_truthy( $add_qr_codes );
 		}
 
-		$preview_context['add_event_links'] = tribe_is_truthy( Arr::get( $vars, 'eventLinks', '' ) );
-		$preview_context['add_ar_fields']   = tribe_is_truthy( Arr::get( $vars, 'arFields', '' ) );
+		$preview_context['add_event_links']    = tribe_is_truthy( Arr::get( $vars, 'eventLinks', '' ) );
+		$preview_context['add_ar_fields']      = tribe_is_truthy( Arr::get( $vars, 'arFields', '' ) );
+
+		$rsvp_using_ticket_email               = tribe_is_truthy( Arr::get( $vars, 'useTicketEmail', '' ) );
+		if ( $rsvp_using_ticket_email ) {
+			$preview_context['using_ticket_email'] = true;
+		}
```

Comment:
Why are we using `$rsvp_using_ticket_email` when we can just use `$preview_context['using_ticket_email']`?
```suggestion
		$preview_context['using_ticket_email'] = tribe_is_truthy( Arr::get( $vars, 'useTicketEmail', '' ) );
```